UtahUITests/SchedulerAndQuestionnaireTests.swift

```diff
@@ -32,18 +32,35 @@ class SchedulerAndQuestionnaireTests: XCTestCase {
         XCTAssertTrue(app.staticTexts["Please complete this task once a month."].waitForExistence(timeout: 2))
         XCTAssertTrue(app.staticTexts["Start Task"].waitForExistence(timeout: 2))
         app.staticTexts["Start Task"].tap()
+        
+        try app.testSurveyLogic()
+    }
+    
+    func testVideoDemo() throws {
+        let app = XCUIApplication()
+        
+        XCTAssertTrue(app.tabBars["Tab Bar"].buttons["Questions"].waitForExistence(timeout: 2))
+        app.tabBars["Tab Bar"].buttons["Questions"].tap()
 
-        try app.navigateVenousQuestionnaire()
+        XCTAssertTrue(app.buttons["Video Demo"].waitForExistence(timeout: 2))
+        app.buttons["Video Demo"].tap()
+        
+        XCTAssertTrue(app.staticTexts["Demonstration Below"].waitForExistence(timeout: 2))
     }
 }
 
 extension XCUIApplication {
-    func navigateVenousQuestionnaire() throws {
```

Comment:
I think the merge from main messed this file up a bit, any chance we can revert it back to what main had and then just add the test for the video as a separate func? My PR I'm working on now is based more off main so might help in the future too